depthai_ros_driver/include/depthai_ros_driver/device_ros.hpp

```diff
@@ -188,7 +188,7 @@ class DeviceROS : public dai::DeviceBase {
                 msg.data = reader.bufferData();
 
                 // publish data
-                pub.publish(adapt_dai2ros<MsgType>::convert(msg));
+                pub.publish(adapt_dai2ros<MsgType>::convert(msg, name));
```

Comment:
`frame_id` is coordinate something like `depthai_optical_frame`.
but yes, it's hard to get the `frame_id` from elsewhere...

Reply:
I think we can do name + "_optical_frame" for the image

Reply:
sounds good. maybe `depthai_` + name + `_optical_frame` in case of `depthai_datatype_msgs::RawImgFrame` but that can depend on the `InputType`

Reply:
depthai_ prefix is unneeded because we can add `depthai_<id>/` as the nodehandle prefix

Reply:
I believed this is resolved by kunal's latest commit